remote/style.css

```diff
@@ -97,13 +97,103 @@ body > .is-root-container,
 	 padding-left: var(--wp--custom--spacing--outer);
  }
 
+/* Button styles */
+.wp-block-search__button,
+.wp-block-button__link,
+.wp-block-search__button {
+	border-radius: var(--wp--custom--button--border--radius);
+	background-color: var(--wp--preset--color--primary);
+	color: var(--wp--preset--color--background);
+	border: 1px solid var(--wp--preset--color--primary);
+}
 
 /*
-* Needed until https://github.com/WordPress/gutenberg/issues/29167 is addressed.
-*/
-.wp-block-post-comments input[type=submit].wp-block-button__link {
-	border-radius: var(--wp--custom--button--border--radius);
+ * Button hover styles.
+ * Necessary until the following issue is resolved in Gutenberg:
+ * https://github.com/WordPress/gutenberg/issues/27075
+ */
+
+ :is(
+	 .wp-block-search__button,
+	 .wp-block-button__link,
+	 .wp-block-search__button
+):is(
+	:hover,
+	:active,
+	:focus
+) {
 	border: 1px solid var(--wp--preset--color--foreground);
-	background-color: transparent;
+	background-color: var(--wp--preset--color--foreground);
+	color: var(--wp--preset--color--background);
+	cursor: pointer;
+}
+
+:is(
+	 .wp-block-search__button,
+	 .wp-block-button__link,
+	 .wp-block-search__button
+):is(
+	:hover,
+	:active,
+	:focus
+) {
+	border: 1px solid var(--wp--preset--color--foreground);
+	background-color: var(--wp--preset--color--foreground);
+	color: var(--wp--preset--color--background);
+	cursor: pointer;
+}
+
+:is(
+	 .wp-block-search__button,
+	 .wp-block-button__link,
+	 .wp-block-search__button
+):focus {
+	 outline: 1.5px dotted var(--wp--preset--color--foreground);
+	 outline-offset: 3px;
+}
+
+/*
+ * Link Details
+ */
+
+a {
 	color: var(--wp--preset--color--foreground);
+	text-decoration-thickness: 0.075ex;
+	text-underline-offset: 0.125em;
+	text-decoration: underline;
+}
+
+.wp-block-site-title a {
+	text-decoration: none;
+}
+
+:is(
+	a,
+	.wp-block-site-title,
+	.wp-block-post-title,
+	.wp-block-post-date
+):hover {
+	color: var(--wp--preset--color--primary);
+	text-decoration: none;
+}
+
+:is(
+	a,
+	.wp-block-site-title,
+	.wp-block-post-title,
+	.wp-block-post-date
+):active {
+	color: var(--wp--preset--color--primary);
+	text-decoration: underline;
+}
+
+:is(
+	a,
```

Comment:
I think this selector is unnecessarily affecting the site title on hover + active:

![Screen Shot 2022-02-23 at 11 21 57 AM](https://user-images.githubusercontent.com/5375500/155392160-9e6f382f-e883-44d1-a351-3d496f828f39.png)

Reply:
This seems to be fixed with the last commit